wgpu-native/Cargo.toml

```diff
@@ -37,10 +37,10 @@ rendy-descriptor = "0.4"
 serde = { version = "1.0", features = ["serde_derive"], optional = true }
 vec_map = "0.8"
 
-[target.'cfg(unix)'.dependencies]
+[target.'cfg(all(unix, not(target_os = "ios"), not(target_os = "macos")))'.dependencies]
 gfx-backend-vulkan = { version = "0.3.0", features = ["x11"] }
 
-[target.'cfg(not(unix))'.dependencies]
+[target.'cfg(any(not(unix), target_os = "ios", target_os = "macos"))'.dependencies]
```

Comment:
We shouldn't need to build gfx-backend-vulkan on macOS or iOS if it will never be selected as the adapter, or do we want to purposely keep it?

Reply:
I think you are right, we need to be more consistent here with Vulkan: either allow the adapter, or disallow it at compile time... Let's follow-up with this.